web-app/src/router/routes.ts

```diff
@@ -12,20 +12,20 @@ const routes: RouteConfig[] = [
         meta: { public: true },
       },
       {
-        path: 'tournamentsList',
+        path: 'tournaments',
         name: 'TournamentsList',
         component: () => import('src/pages/tournament/TournamentsList.vue'),
         meta: { public: true }, //TODO change to false
       },
       {
-        path: 'tournament/:id',
+        path: 'tournaments/:id',
         name: 'TournamentDetails',
         // component: () => import('pages/tournament/Details.vue'),
         component: () => import('pages/Index.vue'),
         meta: { public: true }, // TODO: change to false
```

Comment:
Here should be `tournament/:id`

Reply:
No it shouldn't

Reply:
maybe `should` is bad word. It was only suggestion. look for example:
`https://github.com/Zummek/best-tournament-app/pull/13`
`https://github.com/Zummek/best-tournament-app/pulls`